File: 31/Furgone.py
from Veicoli import Veicoli 
class Furgone(Veicoli):
    def __init__(self, codice, tipo, targa, modello, marca, costoGiornaliero,rimorchio,numPosti,vanoCarico) -> None:
        super().__init__(codice, tipo, targa, modello, marca, costoGiornaliero)
        self.rimorchio = rimorchio
        self.numPosti = numPosti
        self.vanoCarico = vanoCarico

    print("tipo, targa, codice, modello, marca, costo giornaliero,bagagliaio, cilindrata, categoria, rimorchio, numero di posti,vano di carico")
        
    def __str__(self) -> str:
        return self.tipo+"\t"+self.targa+"\t"+str(self.codice)+"\t"+self.modello+"\t"+self.marca+"\t"+str(self.costoGiornaliero)+"\t"+"\t-"+"\t-"+"\t-"+self.rimorchio+"\t"+str(self.numPosti)+"\t"+self.vanoCarico

File: sim07/src/Furgone.java
public class Furgone extends Veicolo {
    String categoria;
    int num_posti;
    Boolean vanno;
    public Furgone(int codice, String tipo, String targa, String modello, String marca, String categoria, int num_posti,
            Boolean vanno) {
        super(codice, tipo, targa, modello, marca);
        this.categoria = categoria;
        this.num_posti = num_posti;
        this.vanno = vanno;
    }
    @Override
    public String toString() {
        return tipo+targa+codice+modello+marca+costo_giornaliero+" - "+" - "+categoria+num_posti+vanno;
    }
}
